docker/dev.docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.1'
+services:
+  pgdatabase:
+    image: postgres:13
```

Comment:
On the production instance, Postges 13 is running. However, the current docker compose file uses 14.2.
I'm a fan of keeping these versions in sync. If we merge it like this, I think renovate will pick it up in the next time anyways